app/models.py

```diff
@@ -227,23 +227,25 @@ class Goals(db.Model):
     goal_category = db.Column(db.String(64), db.ForeignKey('categories.category_name'))
     description = db.Column(db.String(64))
     num_of_steps = db.Column(db.Integer)
+    steps = db.relationship("Steps", cascade="all,delete", backref="goals", passive_deletes=True)
```

Comment:
"all,delete" is redundant here. Did you mean to type "delete-orphan"?
See documentation here: http://docs.sqlalchemy.org/en/latest/orm/cascades.html

Reply:
using delete-orphan may be dangerous in this case because it also deletes children which have been disconnected from the parent element. So say there is a step that was accidentally assigned to one goal, then it is changed to another, I believe delete-orphan would delete the child even if the old goal was deleted. It would be good, however, to look more into cascades because they will definitely come up when we are working with linking the member goals. I'll merge it as is, but definitely do more research when making the member goals functions.